launch/online_navtech_warthog.launch.yaml

```diff
@@ -0,0 +1,46 @@
+## Online Lidar VTR3
+session_name: vtr_online_navtech_warthog
+
+environment:
+  # CYCLONEDDS_URI: ${VTRSRC}/config/cyclonedds.default.xml # custom dds configs
+  ROS_DOMAIN_ID: "7"  # set this to a unique number when multiple ROS2 dependent system running on the same network
```

Comment:
What about the discovery servers?

Reply:
I am testing offline so it is more convenient for me to turn them off atm